datafusion/core/src/lib.rs

```diff
@@ -488,16 +488,16 @@
 //! DataFusion automatically runs each plan with multiple CPU cores using
 //! a [Tokio] [`Runtime`] as a thread pool. While tokio is most commonly used
 //! for asynchronous network I/O, the combination of an efficient, work-stealing
-//! scheduler and first class compiler support for automatic continuation
-//! generation (`async`), also makes it a compelling choice for CPU intensive
+//! scheduler, and first class compiler support for automatic continuation
+//! generation (`async`) also makes it a compelling choice for CPU intensive
 //! applications as explained in the [Using Rustlang’s Async Tokio
 //! Runtime for CPU-Bound Tasks] blog.
 //!
 //! The number of cores used is determined by the `target_partitions`
 //! configuration setting, which defaults to the number of CPU cores.
 //! While preparing for execution, DataFusion tries to create this many distinct
 //! `async` [`Stream`]s for each `ExecutionPlan`.
-//! The `Stream`s for certain `ExecutionPlans`, such as as [`RepartitionExec`]
+//! The `Stream`s for certain `ExecutionPlan`s, such as [`RepartitionExec`]
```

Comment:
```suggestion
//! The `Stream`s for certain [`ExecutionPlan`]s, such as [`RepartitionExec`]
```

Reply:
Thanks for pointing that out. I made your suggested change and also added some other missing intra-doc links as well.